enterprise/backend/src/metabase_enterprise/transforms/schema.clj

```diff
@@ -0,0 +1,68 @@
+(ns metabase-enterprise.transforms.schema
+  (:require
+   [metabase.lib.metadata.column :as lib.metadata.column]
+   [metabase.util.malli.registry :as mr]
+   [metabase.util.malli.schema :as ms]))
+
+(mr/def ::keyset-strategy
+  [:map
+   [:type [:= "keyset"]]
+   [:keyset-column :string]
+   ;; required for mbql and python transforms
+   [:keyset-filter-unique-key {:optional true} ::lib.metadata.column/column-unique-key]
+   ;; required for python transforms
```

Comment:
i wonder if it would be better to allow it for MBQL transforms too, to avoid having to manually add the limit to the MBQL query